### ISSUE TITLE
Feature Request: Auto name based on SVG attribute ID

### ISSUE THREAD

**Issue by jbutkowski** [rossmoody/svg-gobbler#164]
Manually naming large collections is tedious. Having an auto naming option would be an awesome feature.

**Comment by rossmoody** [comment]
I appreciate the feedback very much. So I understand, how are you renaming the collections currently that makes it tedious? There is the inline renaming, and I'd like to add a right-click context menu to the sidebar, but how were you thinking about it?

<img width="1273" alt="Screen Shot 2024-01-01 at 9 08 45 AM" src="https://github.com/rossmoody/svg-gobbler/assets/29072694/d0d8c1a2-b853-4858-93d4-d7672a71124d">

**Comment by rossmoody** [comment]
@jbutkowski Do you mean naming the collection itself or the SVGs inside the collection when downloading as a group?

**Comment by jbutkowski** [comment]
@rossmoody No, not the collection. The individual SVGs. Naming a few SVG's is fine - but a large collection is not a lot of fun. 😅

**Comment by rossmoody** [comment]
5.6 added functionality to prefix filenames manually in an ascending order. This can be improved in future updates, but I was having a hard time dynamically determining a good file name based upon unknown SVG metadata like an ID, so I opted to have it manually set in the export panel via a text input.
